resources/assets/js/app/views/comments/TopicCommentNew.js

```diff
@@ -0,0 +1,208 @@
+var Marionette = require('backbone.marionette');
+var logger = require('../../instances/logger');
+var _ = require('underscore');
+var Radio = require('backbone.radio');
+var Dropzone = require('dropzone');
+var currentUser = require('../../initializers/currentUser');
+var AttachmentModel = require('../../models/AttachmentModel');
+
+module.exports = Marionette.ItemView.extend({
+    template: 'TopicCommentNew',
+    _dropZone: null,
+    _files: [],
+
+    ui: {
+        'submit': '#submit',
+        'close': '.close',
+        'errors': '.errors',
+        'loader': '.loader',
+        'commentDlg': '#commentdlg'
+    },
+
+    modelEvents: {
+        'invalid': function (model, errors, options) {
+            //logger('model is invalid', errors, options);
+            this.showLoader(false);
+            var ui = this.ui;
+            ui.errors.empty();
+            errors.message.forEach(function (message, i) {
+                ui.errors.append(message);
+            });
+            this.showErrors(true);
+        },
+    },
+
+    events: {
+        'click @ui.submit': 'submitComment',
+    },
+
+    showLoader: function (show) {
+        return (show ? this.ui.loader.removeClass('hidden') : this.ui.loader.addClass('hidden'));
```

Comment:
toggleClass will change state on opposite value, this method and logic in it is ok